ember-app/app/initializers/sockets.js

```diff
@@ -6,12 +6,13 @@ export function initialize(container, application){
   let socket = Ember.Object.extend({
     correlationId : correlationId,
     sockets: {},
+    publish: Ember.K,
     subscribe: Ember.K,
     unsubscribe: Ember.K,
     subscribeTo: Ember.K
   });
 
-  if(application.get("socketBackend")){
+  if(application.get("socketBackend") &&  window.Faye){
```

Comment:
Yeah, I'd check for `Faye` before we use it [here](https://github.com/huboard/huboard-web/blob/47242aba93d1b7641e599ca57e0f69e55def8b77/ember-app/app/utilities/messaging/fayeClient.js#L5).